pkg/webhook/validation.go

```diff
@@ -27,11 +27,11 @@ import (
 	"knative.dev/pkg/apis"
 	"knative.dev/pkg/logging"
 
-	"github.com/sigstore/policy-controller/cmd/cosign/cli/fulcio/fulcioroots"
+	"github.com/sigstore/cosign/cmd/cosign/cli/fulcio/fulcioroots"
```

Comment:
This seems to be the only cmd/cosign dependency, and it's moving to sigstore/sigstore in https://github.com/sigstore/sigstore/pull/435 🎉

Reply:
Yeah, can't wait!